pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1282,6 +1289,50 @@ func triggerPublishDevInfo(ctxPtr *zedagentContext) {
 	}
 }
 
+func triggerPublishAllInfo(ctxPtr *zedagentContext) {
```

Comment:
How about adding a comment that we run a goroutine since all the publish can take a long time and the goroutine will block sending on the TriggerInfo channel?